allure-junit5/src/test/java/io/qameta/allure/examples/junit5/AllureDescriptionTest.java

```diff
@@ -0,0 +1,27 @@
+package io.qameta.allure.examples.junit5;
+
+import io.qameta.allure.Allure;
+import io.qameta.allure.Description;
+import org.junit.jupiter.api.Test;
+
+public class AllureDescriptionTest {
+
+    @Test
+    @Description("Static description")
+    public void annotationDescriptionTest() {
+    }
+
+    /**
+     * JavaDoc description
+     */
+    @Test
+    @Description(useJavaDoc = true)
```

Comment:
by the way, in Python the "use docstring" behaviou is the default one... Maybe we should think about more consistent allure behaviour among different languages...